src/main/java/com/minecolonies/colony/materials/QuantityNotFound.java

```diff
@@ -7,6 +7,11 @@
  */
 class QuantityNotFound extends RuntimeException
 {
+
+    static final long serialVersionUID = -703925360745766939L;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Explicitly declare the visibility for "serialVersionUID". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2039)